### PR TITLE
Event/get annotations service

### PR DESCRIPTION
**Pull request by Andreas-Garcia**
Add GET events/{uri}/annotations service
Remove annotations from GET events/{uri} service

### CODE REVIEW

File: phis2-ws/src/main/java/opensilex/service/dao/ConcernedItemDAO.java [hunk 1]
```diff
@@ -91,8 +93,15 @@ public static void prepareQueryWithConcernedItemFilters(SPARQLQueryBuilder query
         }
     }
     
+    /**         
+            if (searchConcernedItemUri != null) {
+                query.appendAndFilter("regex (str(" + CONCERNED_ITEM_URI_SELECT_NAME_SPARQL + ")" + ", \"" + searchConcernedItemUri + "\", \"i\")");
+            }
+        }
+    }
+
```
Comment (annetireau) [inline phis2-ws/src/main/java/opensilex/service/dao/ConcernedItemDAO.java]: ?

File: phis2-ws/src/main/java/opensilex/service/resource/dto/event/EventDTO.java [hunk 2]
```diff
@@ -24,20 +24,19 @@
...
     
     /**
-     * SILEX:info
-     * "type" can not be used as a field name in DTOs due to XML interpretation
-     * issues.
+     * //SILEX:info
+     * "type" can not be used as a field name in DTOs due to XML interpretation issues.
      * @see https://stackoverflow.com/questions/33104232/eclipselink-missing-class-for-indicator-field-value-of-typ
-     * SILEX:info
+     * //\
```
Comment (annetireau) [inline phis2-ws/src/main/java/opensilex/service/resource/dto/event/EventDTO.java]: Add final SILEX tag

File: phis2-ws/src/main/java/opensilex/service/dao/ConcernedItemDAO.java (hunk 1 above)
Comment (Andreas-Garcia) [reply phis2-ws/src/main/java/opensilex/service/dao/ConcernedItemDAO.java]: I remove this useless comment

Comment (Andreas-Garcia) [reply phis2-ws/src/main/java/opensilex/service/dao/ConcernedItemDAO.java]: I remove this useless comment

**Pull request merged**